src/extractor/guidance/turn_handler.cpp

```diff
@@ -773,17 +774,18 @@ void TurnHandler::handleDistinctConflict(const EdgeID via_edge,
         return;
     }
 
-    if (getTurnDirection(left.angle) == DirectionModifier::Right)
+    // turn to the right
+    if (getTurnDirection(left.angle) <= 180)
     {
         if (angularDeviation(left.angle, 85) >= angularDeviation(right.angle, 85))
         {
-            left.instruction = {left_type, DirectionModifier::Right};
-            right.instruction = {right_type, DirectionModifier::SharpRight};
+            left.instruction = {left_type, DirectionModifier::SlightRight};
```

Comment:
I notices this bug in the process of fixing the above.
`Left` is farther away from a 90 degree turn than `right`. So left cannot be a stronger turn than `right`.
These two were mixed up.